tests/recipes/click-test-feedstock/ci_support/matrix/README

```diff
@@ -0,0 +1 @@
+This file is automatically generated by conda-smithy.  To change any matrix elements, you should change conda-smithy's input conda_build_config.yaml and re-render the recipe, rather than editing these files directly.
```

Comment:
Format comment: Would it be alright if we wrap this text a bit. Naively 79 characters tends to be a good limit. Though if you have preferences, feel free to suggest. :)

Reply:
Should add I know this is just a test file. Just first one I found. Can make these changes to the actual copy.

Reply:
Also would be good if it got a `.txt` extension or other if it should be fancy formatted.